### PR TITLE
Regenerate gapic clients, add IoT and Web Security Scanner clients

### PR DESCRIPTION
**Pull request by vam-google**
Also remove beta packages for dlp, as they are removed from googleapis. IoT packages were moved from `cloudiot` to just `iot`.

### CODE REVIEW

File: google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ImageAnnotatorClient.java [hunk 1]
```diff
@@ -217,6 +232,117 @@ private final BatchAnnotateImagesResponse batchAnnotateImages(
...
+   * try (ImageAnnotatorClient imageAnnotatorClient = ImageAnnotatorClient.create()) {
+   *   List&lt;AsyncAnnotateFileRequest&gt; requests = new ArrayList&lt;&gt;();
+   *   AsyncBatchAnnotateFilesResponse response = imageAnnotatorClient.asyncBatchAnnotateFilesAsync(requests).get();
+   * }
+   * </code></pre>
+   *
+   * @param requests Individual async file annotation requests for this batch.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  public final OperationFuture<AsyncBatchAnnotateFilesResponse, OperationMetadata>
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ImageAnnotatorClient.java]: Well crap I was wondering when this might happen. We have a GA client which formerly only used stable features from GAX, but now the API has added an API method which uses LRO, which isn't stable yet. We need to mark this `@BetaApi` for the moment - I believe we have a visibility field in GAPIC config that can do this.

File: google-cloud-clients/google-cloud-texttospeech/README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Versioning
 
 This library follows [Semantic Versioning](http://semver.org/).
 
-It is currently in major version zero (``0.y.z``), which means that anything may change at any time and the public API should not be considered stable.
+It is currently in major version zero (``0.y.z``), which means that anything may change at any time and the public API should not be considered stable.d
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-texttospeech/README.md]: undo this change

File: google-cloud-clients/google-cloud-dlp/pom.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@
     <version>0.47.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-clients:current} -->
   </parent>
   <properties>
-    <site.installationModule>google-cloud-dlp-v2beta1</site.installationModule>
+    <site.installationModule>google-cloud-dlp-v2</site.installationModule>
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-dlp/pom.xml]: wait this doesn't seem right - shouldn't this match the artifact? (`google-cloud-dlp`)

File: google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ImageAnnotatorClient.java (hunk 1 above)
Comment (vam-google) [reply google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ImageAnnotatorClient.java]: Added `@BetaApi` annotation here manually, and posted [gapic-generator PR](https://github.com/googleapis/gapic-generator/pull/2004) to add it on generation phase for all long-running operations in java.

**Comment by vam-google** [comment]
@garrettjonesgoogle PTAL

**Comment by vam-google** [comment]
@garrettjonesgoogle PTAL
